docs/tools/streamsets.txt

```diff
@@ -10,168 +10,159 @@ Building Data Stream pipelines with CrateDB and StreamSets Data Collector
-   and configure the CrateDB JDBC Driver loading as it was done for the JDBC
-   destination configuration. In the JDBC tab of the CrateDB JDBC origin we set
-   the connection string (1), the SQL query for fetching the data from the
-   database (2), the offset value and column (3). For more detail information
-   on how to set the offset value and column see the StreamSets `documentation
-   <https://streamsets.com/documentation/datacollector/1.1.3/help/#Origins/JDBCConsumer.html#concept_nxz_2kz_bs>`_.
-
-   .. image:: 3-streamsets-source-jdbc.png
+the data stream pipeline. As sample data, we use the `AMPLab rankings dataset`_.
+The data can be imported from `AWS S3`_ to the CrateDB database using prepared
+data import queries hosted in the `CrateDB demo repository`_. Create the
+`rankings table`_ and import the `rankings data`_. In the demo, we use a
+dataset that contains 18 million of records. Having the CrateDB cluster set up
+and the rankings sample data is imported, we can start building the data stream
+pipeline for streaming the data from the CrateDB database to CSV files.
+
+1. Create a `new data collector pipeline`_ and configure the CrateDB JDBC Driver
+   loading as it was done for the JDBC destination configuration. In the JDBC
+   tab of the CrateDB JDBC origin we set the connection string, the SQL query
+   for fetching the data from the database, the initial offset, and offset
+   column. For more detail information on how to set the offset column and its
+   value, see the `JDBC Query Consumer offset`_ documentation.
+
+   .. image:: 4-streamsets-source-jdbc-conf.png
       :alt: Configure the CrateDB JDBC driver
 
    **IMPORTANT**
 
-   To avoid unsupported transaction setting queries that may be invoked
-   against CrateDB, it is necessary to uncheck **Enforce Read-only
-   Connection** on the advanced tab of the JDBC consumer.
+   To avoid unsupported transaction setting queries that may be invoked against
+   CrateDB, it is necessary to uncheck ``Enforce Read-only Connection`` on the
+   advanced tab of the JDBC consumer.
 
 2. We stream the records from CrateDB to CSV files. In order to accomplish that
    we provide the path where the files are going to be created and set the
    output file format in the directory origin to *Delimited*.
 
-   .. image:: 4-streamsets-source-fs-conf1.png
+   .. image:: 4-streamsets-source-fs-conf.png
       :alt: Configure CSV destination
 
 3. Now we can start the pipeline and see rankings data streaming statistics
    form the CrateDB database to CSV files.
 
-   .. image:: 6-streamsets-origin-report.png
+   .. image:: 4-streamsets-origin-report.png
       :alt: CrateDB to CSV report
 
--  `Download StreamSets <https://streamsets.com/opensource/>`_
--  `StreamSets Official documentation
-   <https://streamsets.com/documentation/datacollector/latest/help/>`_
+
+.. _`CrateDB`: https://crate.io
```

Comment:
please sort this whole list alphabetically

Reply:
please also drop the ` from around the link titles. they're not needed in the link list